dev/hello-world.config.yaml

```
@@ -1,4 +1,5 @@
 db_dsn: postgres://dev:dev@localhost:5432/dev
+max_conn_size: 32
```

Adding a namespace can reduce ambiguity
```suggestion
meta:
   max_conn_size: 32
```